modules/core/src/main/scala/com/github/scoquelin/arugula/commands/LettuceRedisStringAsyncCommands.scala

```diff
@@ -1,21 +1,57 @@
 package com.github.scoquelin.arugula.commands
 
+import scala.collection.immutable.ListMap
 import scala.concurrent.Future
 import scala.concurrent.duration.FiniteDuration
+import scala.jdk.CollectionConverters._
+
 import com.github.scoquelin.arugula.internal.LettuceRedisCommandDelegation
+import io.lettuce.core.{GetExArgs, KeyValue}
 
 import java.util.concurrent.TimeUnit
 
 private[arugula] trait LettuceRedisStringAsyncCommands[K, V] extends RedisStringAsyncCommands[K, V] with LettuceRedisCommandDelegation[K, V] {
+
+  override def append(key: K, value: V): Future[Long] =
+    delegateRedisClusterCommandAndLift(_.append(key, value)).map(Long2long)
+
   override def get(key: K): Future[Option[V]] =
     delegateRedisClusterCommandAndLift(_.get(key)).map(Option.apply)
 
   override def getDel(key: K): Future[Option[V]] =
     delegateRedisClusterCommandAndLift(_.getdel(key)).map(Option.apply)
 
+  override def getEx(key: K, expiresIn: FiniteDuration): Future[Option[V]] =
+    (expiresIn.unit match {
+      case TimeUnit.MILLISECONDS | TimeUnit.MICROSECONDS | TimeUnit.NANOSECONDS =>
+        delegateRedisClusterCommandAndLift(_.getex(key, GetExArgs.Builder.ex(expiresIn.toMillis)))
+      case _ =>
+        delegateRedisClusterCommandAndLift(_.getex(key, GetExArgs.Builder.ex(java.time.Duration.ofSeconds(expiresIn.toSeconds))))
+    }).map(Option.apply)
+
+  override def getRange(key: K, start: Long, end: Long): Future[Option[V]] =
+    delegateRedisClusterCommandAndLift(_.getrange(key, start, end)).map(Option.apply)
+
   override def getSet(key: K, value: V): Future[Option[V]] =
     delegateRedisClusterCommandAndLift(_.getset(key, value)).map(Option.apply)
 
+  override def mGet(keys: K*): Future[ListMap[K, Option[V]]] =
+    delegateRedisClusterCommandAndLift(_.mget(keys: _*)).map {
+      case null => ListMap.empty
+      case kvs => ListMap(kvs.toArray.collect {
+        case kv if kv.isInstanceOf[KeyValue[_, _]] =>
+          val keyValue = kv.asInstanceOf[KeyValue[K, V]]
+          if (keyValue.hasValue) keyValue.getKey -> Some(keyValue.getValue)
+          else keyValue.getKey -> None
+      }: _*)
+    }
```

Comment:
```suggestion
    delegateRedisClusterCommandAndLift(_.mget(keys: _*)).map {
      case null => ListMap.empty
      case kvs => ListMap.from(kvs.asScala.collect {
        case keyValue =>
          if (keyValue.hasValue) keyValue.getKey -> Some(keyValue.getValue) else keyValue.getKey -> None
      })
    }
```

Reply:
yeah I was just hoping to avoid issues with sort order changing w the mGet but yeah we might not need to. It seemed simple enough to preserve the sort order with this approach? but I don't want to make it confusing.